fastest.md

```diff
@@ -0,0 +1,51 @@
+Introduction
+============
+
+[fastest](https://github.com/liuggio/fastest) is a tool for executing PHPUnit tests in parallel.
+Once LiipFunctionalTestBundle has been configured, it's easy to use fastest.
+
+Installation
+============
+
+1. Install paratest: `composer require "liuggio/fastest=~1.4"`
```

Comment:
It has been fixed after this PR was merged: https://github.com/liip/LiipFunctionalTestBundle/pull/283